app/src/main/java/org/amfoss/templeapp/activities/DeleteData.java

```diff
@@ -19,100 +19,100 @@
-  RelativeLayout totalLayout;
-  int flag;
-  private Button delete;
-  private EditText uid1ET;
-
-  protected void onCreate(Bundle savedInstanceState) {
-    super.onCreate(savedInstanceState);
-    setContentView(R.layout.delete_data);
-    delete = (Button) findViewById(R.id.delete_btn);
-    uid1ET = (EditText) findViewById(R.id.uid);
-
-    radioGroup = (RadioGroup) findViewById(R.id.radiogroup);
-    totalLayout = (RelativeLayout) findViewById(R.id.total_View);
-
-    radioGroup.setOnCheckedChangeListener(
-        new RadioGroup.OnCheckedChangeListener() {
-          public void onCheckedChanged(RadioGroup group, int checkedId) {
-            switch (checkedId) {
-              case R.id.radio_donate:
-                totalLayout.setVisibility(View.VISIBLE);
-                flag = 0;
-                Toast.makeText(
-                        getBaseContext(), getString(R.string.delete_donated), Toast.LENGTH_LONG)
-                    .show();
-                break;
-              case R.id.radio_pooja:
-                totalLayout.setVisibility(View.VISIBLE);
-                flag = 1;
-                Toast.makeText(
-                        getBaseContext(), getString(R.string.delete_pooja), Toast.LENGTH_LONG)
-                    .show();
-                break;
-            }
-          }
-        });
-
-    delete.setOnClickListener(
-        new View.OnClickListener() {
-          @Override
-          public void onClick(View view) {
-
-            if (flag == 1) {
-              id = getString(R.string.REG) + uid1ET.getText().toString();
-            } else {
-              id = getString(R.string.DON) + uid1ET.getText().toString();
-            }
-
-            new DeleteDataActivity().execute();
-          }
-        });
-  }
-
-  class DeleteDataActivity extends AsyncTask<Void, Void, Void> {
+    String id;
+    String name;
+    RadioGroup radioGroup;
+    RelativeLayout totalLayout;
+    int flag;
+    private Button delete;
+    private EditText uid1ET;
+
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.delete_data);
+        delete = (Button) findViewById(R.id.delete_btn);
+        uid1ET = (EditText) findViewById(R.id.uid);
+
+        radioGroup = (RadioGroup) findViewById(R.id.radiogroup);
+        totalLayout = (RelativeLayout) findViewById(R.id.total_View);
+
+        radioGroup.setOnCheckedChangeListener(
+                new RadioGroup.OnCheckedChangeListener() {
+                    public void onCheckedChanged(RadioGroup group, int checkedId) {
+                        switch (checkedId) {
+                            case R.id.radio_donate:
+                                totalLayout.setVisibility(View.VISIBLE);
+                                flag = 0;
+                                Toast.makeText(
+                                                getBaseContext(), getString(R.string.delete_donated), Toast.LENGTH_LONG)
```

Comment:
This seems like too much indentation. I think 2 space indentation is good enough.

Reply:
Those has to be in same line. I guess that is the reason it is giving that big indentation

Reply:
Those has to be in same line. I guess that is the reason it is giving that big indentation

Reply:
Yeah, I saw other changes too. I thought that 4 tabs are too much. According to me, 2 are enough.